ECHomepage/php/sendMail.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+function sanitize_my_email($field) {
+    $field = filter_var($field, FILTER_SANITIZE_EMAIL);
+    if (filter_var($field, FILTER_VALIDATE_EMAIL)) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+function IsInjected($str)
+{
+    $injections = array('(\n+)',
+           '(\r+)',
+           '(\t+)',
+           '(%0A+)',
+           '(%0D+)',
+           '(%08+)',
+           '(%09+)'
+           );
+
+    $inject = join('|', $injections);
+    $inject = "/$inject/i";
+
+    if(preg_match($inject,$str))
+    {
+      return true;
+    }
+    else
+    {
+      return false;
+    }
+}
+
+$to      = 'leitung@ec-hormersdorf.de';
```

Comment:
in der testphase sollten wir es auf unsere mail binden sonst bekommt christopher spam :D